pkg/controller/imagepulljob/imagepulljob_controller.go

```diff
@@ -304,15 +279,10 @@ func (r *ReconcileImagePullJob) syncNodeImages(job *appsv1alpha1.ImagePullJob, n
 				if tagSpec.Tag != imageTag {
 					continue
 				}
-				if containsObjectRef(tagSpec.OwnerReferences, *ownerRef) {
+				if utilimagejob.FormatNodeImageTagSpecIfHas(tagSpec, imageTag, ownerRef, now) {
```

Comment:
I prefer the old logic that update tagSpec here, which can be easily compared with the not-found initialization below. `FormatNodeImageTagSpecIfHas` is a little confusing.

Moving containsObjectRef to util is fine.

Reply:
Agree.